tools/SchemaValidation/imodel-native-tests.yaml

```diff
@@ -70,25 +73,25 @@ stages:
       demands: Agent.OS -equals Windows_NT
 
     variables:
-      BB_V: 4
+      BB_V: '-v 4'
       BB_ARCH: x64
       PY_EXE: py -3
       BB_SRC_ROOT: $(Agent.BuildDirectory)/bbsrc
-      bb: $(BB_SRC_ROOT)/BentleyBuild/BentleyBuild.py
+      BB: $(BB_SRC_ROOT)/BentleyBuild/BentleyBuild.py
       Branch_Name: $[ stageDependencies.Source_Branch.Find_Branch.outputs['Get_Branch_Name.sourceBranch'] ]
-      strategy: 'iModelCore;SkipPull_imodel-native-internal+iModelJsNodeAddon;SkipPull_imodel-native-internal'
+      STRATEGY: 'iModelCore;SkipPull_imodel-native-internal+iModelJsNodeAddon;SkipPull_imodel-native-internal'
 
     steps:
-    - checkout: imodel-native
     - checkout: imodel-native-internal
 
     - template: build/templates/prebuild.yml@imodel-native-internal
       parameters:
```

Comment:
reWe will only have changes in 'BisSchemas' directory in imodel-native tree, we build the artifacts based on it and we have to validate it with itwin.js-core repository. I think during the discuss you suggested to use the `main` branch for this validation here right?

Reply:
Yeah. Any checkouts to a native branch will be main

Reply:
I mean for `itwin.js-core` repository, we should get it from `main`? and that should serve the purpose for validation that we currently do manually with `imodel02` branch? calling the template `template: build/templates/prebuild.yml@imodel-native-internal` does that validation?

Reply:
No, main branch does not exist for itwinjs-core repo. It would be `imodel02` or `master`. This might be a question for Colin on which to checkout and test against. I don't not know fully what you might need, or what is best right now. What i do know is that I am checking out `imodel02` branch in my pipelines. Calling that template does not do anything with itwinjs-core repo. Pre build only works on native pipeline.

Reply:
If you are doing checkout with `imodel02` branch now then we should also do that. I will try this out.

Adding @ColinKerr so that he is aware of this change. `

Reply:
we should follow the same path that imodel-native does for it's PR validation.  When/if that PR workflow moves away from the imodel02 branch then we should as well.

Reply:
Sounds good!

@DanRod1999 , If you do any further updates do lets us know so that we can keep our changes synchronized. Also right now, it seems like `rush build` is again failing and this part is not stable. Do let us know, when you merge your changes to `main` and make them operational so that we can also do at the same time to get the stability.

Reply:
Now build and test commands are passing so we can merge this and test this job as an optional dependency.